CHANGELOG.md

```diff
@@ -268,7 +269,12 @@ As mentioned a few times in the past, the core team is reviewing the repository
    ```
 
 3. Ensure that you have all the babel dependencies to version `^7.0.0` (you may also need to add `"babel-core": "7.0.0-bridge.0"` as a yarn resolution to ensure retro-compatibility)
-4. If you have a custom packager configuration via `rn-cli.config.js`, you probably need to update it to work with the updated Metro configuration structure (for full detail refer to Metro's [documentation](https://facebook.github.io/metro/docs/en/configuration)); here are some commonly encountered changes to `rn-cli.config.js`:
+4. Upgrading android gradle version to 4.4
+	1. In your project's `android/gradle/wrapper/gradle-wrapper.properties` file, change the `distributionUrl` to `https\://services.gradle.org/distributions/gradle-4.4-all.zip`
+	2. In `android/build.gradle` file add `google()` right above `jcenter()` in both `buildscript` and `allprojects` repositories. Then change Android build tools to version 3.1.3 `classpath 'com.android.tools.build:gradle:3.1.3'`
+	3. In `android/app/build.gradle` file update all your `compile` statements to be `implementation`, e.g. `implementation 'com.facebook.fresco:animated-gif:1.10.0'`
```

Comment:
Can you make sure this is set as "optional"? Your project will work without it.

Reply:
without this change, it causes facebook/react-native#21309 issue in my system

Reply:
@DaniAkash based on @kelset's feedback, it seems like that may be a project specific or environment specific issue; are you open to making the recommended change?

Reply:
I'm ok with not explicitly say to have it optional, can you fix the other issues at least so that we can merge it?

Reply:
Sure.

Reply:
@kelset it's done!